controllers/disruption_controller.go

```diff
@@ -557,134 +727,105 @@ func (r *DisruptionReconciler) handleChaosPodsTermination(instance *chaosv1beta1
 	}
 
 	for _, chaosPod := range chaosPods {
-		removeFinalizer := false
-		ignoreStatus := false
-		target := chaosPod.Labels[chaostypes.TargetLabel]
+		r.terminateChaosPod(instance, chaosPod)
```

Comment:
I think we should rename this function, otherwise this sort of reads like we always terminate every chaos pod in every reconcile loop. I think `maybeTerminateChaosPod`, `handleChaosPodTermination`, etc?

Reply:
yes, fixed